website/docs/index.html.markdown

```diff
@@ -57,8 +57,8 @@ explained below:
 
 ### Environment variables
 
-You can provide your credentials via the `OKTA_ORG_NAME`, `OKTA_BASE_URL`, `OKTA_API_TOKEN`, `OKTA_CLIENT_ID`, 
-`OKTA_API_SCOPES` and `OKTA_PRIVATE_KEY` environment variables, representing your Okta Organization Name, 
+You can provide your credentials via the `OKTA_ORG_NAME`, `OKTA_BASE_URL`, `OKTA_API_TOKEN`, `OKTA_API_CLIENT_ID`, 
+`OKTA_API_SCOPES` and `OKTA_API_PRIVATE_KEY` environment variables, representing your Okta Organization Name, 
```

Comment:
Matching what the code uses:

https://github.com/okta/terraform-provider-okta/blob/6ddf5aa3a874ca8a254fe8d707bf21cc61dc4cf5/okta/provider_test.go#L41

https://github.com/okta/terraform-provider-okta/blob/6ddf5aa3a874ca8a254fe8d707bf21cc61dc4cf5/okta/provider_test.go#L42